### DISCUSSION TITLE
Motion Alpha test phase API telemetry

### DISCUSSION THREAD

**Discussion by TorfinnOlsen** [filecoin-project/motion#46] [General]
During the alpha release testing phase a select number of ISV's will be testing the motion API. Motion team will want to gather some measure of telemetry during that testing phase in order to insure we've recognized any design constraints that need to be revised during Beta, as well as plan for scaling in production.

Some example items we could request ISV's participating in testing to log:

**API performance**
- CPU, Memory demand
- Number of deals/time

**Storage constraints**
- Docker and localDB storage utilization 

**Storage efficiency**
- Storage demand relative to utilization of the API for deal making

**Bandwidth or Throughput constraints**
- Throughput/time

Who can own defining the scope of logging we'll request, collecting that data and then making recommendations for the Beta RC?
-

**Comment by masih** [comment]
We would need to define a more detailed set of metrics.

Assuming the Beta testing is _only_ exposed to a select set of ISVs, one way to do this is is to provide a prometheus endpoint credentials to which the metrics will be pushed to.
